label.py

```diff
@@ -11,80 +11,106 @@
 parser.add_argument("file_name", action="store")
 args = parser.parse_args()
 
-labels = utils.read_labels(labels_directory + args.file_name)
+labels = utils.read_labels(labels_directory + args.file_name + ".csv")
+boundary_boxes = utils.read_boundary_boxes(labels_directory + args.file_name + "_boundary_box.csv")
 
 images_to_show = [i for i in utils.get_images() if i not in labels]
-current_image = None
-
-root = Tk()
-panel1 = Label(root)
-panel1.pack(side="left", padx=10)
-panel2 = Label(root)
-panel2.pack(side="left", padx=10)
-panel3 = Label(root)
-panel3.pack(side="left", padx=10)
+drawing = False
+key_map = {"Escape": 27, "r": 114, "Enter": 13, "y": 121, "n": 110, "d": 100}
+cv2.namedWindow('firefox')
+cv2.namedWindow('chrome')
+
```

Comment:
We should add the window which shows differences back, it was useful to perform labeling more quickly.

Reply:
Are you talking about the third window which was added with overlay among two screenshots?
>@marco-c Regarding #126 I think I should add it in separate PR if you want.
So I will add it in this PR only!

Reply:
Yes, we shouldn't regress that functionality.

Reply:
I will add the functionality after #149 is resolved.